variants/XMC4200/linker_script.ld

```diff
@@ -0,0 +1,281 @@
+/**
+ * @file XMC4200x256.ld
+ * @date 2017-04-20
+ *
+ * @cond
+ *********************************************************************************************************************
+ * Linker file for the GNU C Compiler v1.8
+ * Supported devices: XMC4200-F64x256
+ *                    XMC4200-Q48x256
+ *

```

Comment:
Supported devices is ANY 4200 with 256kB Flash and 40kB RAM, Package is IRRELEVANT to linker